src/components/ProfilePage/SocialLinks.jsx

```diff
@@ -197,7 +208,7 @@ function EditableListItem({
   onClickEmptyContent,
   name,
 }) {
-  const linkDisplay = url != null ?
+  const linkDisplay = (url != null && url !== '') ?
```

Comment:
_.isEmpty() ?

Reply:
I think at this point could is just be `url ? something : somethingElse`
?